proto/cmp/services/seat_map/v4/availability.proto

```diff
@@ -1,5 +1,75 @@
 syntax = "proto3";
 
+// ## SeatMap and SeatMapAvailability Services
+//
+// The SeatMap and SeatMapAvailability services are used to manage and retrieve
+// seat map information and availability for various products, such as flights,
+// trains, cruises and venues for activities like an opera, a concert or a sports
+// event.
+//
+// ### SeatMap general information
+//
+// For smaller applications, such as an airplane, a small concert hall, or an
+// intimate venue, a single, detailed SeatMap is often the most practical choice.
+// In these cases, the entire seating arrangement can be presented clearly in one
+// view without overwhelming the user.
+// A unified map allows passengers or attendees to see exactly where their seat is
+// in relation to entrances, aisles, or the focal point—whether that’s a stage, a
+// screen, or the plane’s layout. Sections can still be used for example for business
+// class or economy class, but the overall map remains relatively small and easy to
+// process.
+//
+// In larger settings like stadiums or convention centers, however, a single
+// detailed map can become overly dense and hard to process due to sheer size.
+// That’s where multiple smaller SeatMaps, grouped by key areas, categories or
+// products, can prove more effective.
+//
+// By focusing each map on specific zones—such as VIP, general admission, or
+// seating blocks can more quickly filter down to the type of product they’re
+// seeking.
+//
+// Still, both strategies have their strengths and trade-offs, which is why a
+// hybrid approach often offers the best balance.
+// This model starts with a single high-level map that shows the full venue at a
+// glance, then allows users to zoom into specific sections for finer detail,
+// using a differentS, instead of drilling down into a subsection of a
+// singleS.
```

Comment:
```suggestion
// This model starts with a single high-level map that shows the full venue at a
// glance, then allows users to zoom into specific sections for finer detail,
// using a different seat map, instead of drilling down into a subsection of a
// single seat map.
```

I guess this section is cropped during a search and replace operation?